### ISSUE TITLE
Focus issues on link dialog

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1700]
When opening the option to add a link within the drive using the keyboard, tabbing through the dialog behaves unexpectedly, preventing proper functionality and use as intended. 
Tabbing through the link input field, a red outline appears, which may be confusing for some users. The red outline should only appear if the link input is empty after the form is submitted, and it should be accompanied by a clear error message. This also links to #1463.
